### PR TITLE
Updated hand joint image in HandTracking.md

### PR DESCRIPTION
**Pull request by CDiaz-MS**
## Overview

An image on our hand tracking documentation was missing the metacarpal joints, the image has been updated to accurately reflect the joints that are tracked.

image

Here is the new image:

ArticulatedHandJoints

### CODE REVIEW

File: Documentation/Input/HandTracking.md [hunk 1]
```diff
@@ -15,8 +15,10 @@ By default the hand joint prefabs are simple geometric primitives. These can be
 > [!WARNING]
 > Avoid using complex scripts or expensive rendering in joint prefabs, since joint objects are transformed on every frame and can have significant performance cost!
 
-<img src="../../Documentation/Images/InputSimulation/MRTK_Core_Input_Hands_JointVisualizerPrefabs.png" height="350px"  style="display:inline;">
```
Comment (keveleigh) [inline Documentation/Input/HandTracking.md:18]: Should this be added with the same file name, so the old image is replaced? Or should we just delete the old image?

Comment (CDiaz-MS) [reply Documentation/Input/HandTracking.md:18]: I added a new pic with a new name so I can delete the old picture

**Pull request merged**